runtime/cudaq/platform/default/rest/helpers/ionq/IonQServerHelper.cpp

```diff
@@ -197,8 +206,15 @@ std::string IonQServerHelper::constructGetResultsPath(std::string &jobId) {
   if (!keyExists("job_path"))
     throw std::runtime_error("Key 'job_path' doesn't exist in backendConfig.");
 
+  // Construct the results path
+  std::string resultsPath = backendConfig.at("job_path") + jobId + "/results";
+
+  // If sharpen is true, add it to the query parameters
+  if (keyExists("sharpen") && backendConfig["sharpen"].get<bool>())
+    resultsPath += "?sharpen=true";
+
   // Return the results path
-  return backendConfig.at("job_path") + jobId + "/results";
+  return resultsPath;
```

Comment:
This code isn't called anywhere, so I suspect this should be added to the `onQServerHelper::constructGetResultsPath(ServerMessage &postResponse)` flavor of this function instead? I'll post suggestions for you to review.

Reply:
True - though I'd like to include sharpen as an option if someone needs to get a job by id. Is it fine to keep sharpen in both methods?